src/sage/schemes/elliptic_curves/ell_finite_field.py

```diff
@@ -1788,7 +1788,10 @@ def twists(self):
             sage: eq = 1728*4*A**3 - j * (4*A**3 + 27*B**2)
             sage: twists2 = []
             sage: for _ in range(10):
-            ....:     V = Ideal([eq, A + B - F.random_element()]).variety()
+            ....:     try:
+            ....:         V = Ideal([eq, A + B - F.random_element()]).variety()
+            ....:     except ValueError:
+            ....:         continue
```

Comment:
```suggestion
            ....:     I = Ideal([eq, A + B - F.random_element()])
            ....:     try:
            ....:         V = I.variety()
            ....:     except ValueError:
            ....:         if I.dimension() == 0:
            ....:              raise
```
Just a little extra protection against inadvertently catching other bugs.